build.gradle

```diff
@@ -23,3 +25,8 @@ subprojects {
         }
     }
 }
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '4.4.1'
```

Comment:
In future we can just bump the version here and run `./gradlew wrapper` to update the Gradle distribution